ecs-deploy/locals.tf

```diff
@@ -31,12 +31,6 @@ locals {
   region            = data.aws_region.current.name
   image_name        = var.app_image_name == null ? "${local.account_id}.dkr.ecr.${local.region}.amazonaws.com/${var.ecr_repo_name}" : var.app_image_name
   logger_image_name = var.with_logger && var.logger_image_name == null ? "${local.account_id}.dkr.ecr.${local.region}.amazonaws.com/${var.logger_ecr_repo_name}" : var.logger_image_name
-  mount_points = var.with_logger ? [
```

Comment:
Should we optionally keep this for projects that have very low request volume, so we don't need to set up Kinesis and all that comes with it? Collecting the logs with vector on the same container is much simpler (it just doesn't scale, which is not important for every project) --> I'm also fine with removing this and setting up Kinesis for all projects 🤷  --- WDYT @samkahchiin

Reply:
Agree. I think we should keep it before we find another easier solution. 
Maintaining kinesis for all projects can be tiring.

Reply:
@swiknaba 

Putting back mountpoint in this commit => 23e13d5